controllers/admin_domains.php

```diff
@@ -384,12 +405,28 @@ public function configuration()
                 $this->DomainsTlds->updateTax($this->post['domains_taxable']);
             }
 
-            $this->flashMessage(
-                'message',
-                Language::_('AdminDomains.!success.configuration_updated', true),
-                null,
-                false
+            // Update cron interval
+            $cron = $this->CronTasks->getTaskRunByKey('domain_tld_synchronization', 'domains');
+            $this->CronTasks->editTaskRun(
+                $cron->task_run_id,
+                [
+                    'interval' => $this->post['domains_sync_frequency'] * 60 * 24,
```

Comment:
Hmmm, we may have an issue here.  The admin/settings/company/automation/ page doesn't actually allow an interval of more than 24 hours.  I think we may need to handle this separate from the cron_task_run by evaluating 'domains_sync_frequency' in the cron job itself and insuring the appropriate number of days have passed.